src/state/slices/opportunitiesSlice/resolvers/thorchainsavers/index.ts

```diff
@@ -16,6 +16,26 @@ import type {
 } from '../../types'
 import type { OpportunitiesMetadataResolverInput } from '../types'
 
+type MidgardPoolResponse = {
```

Comment:
Move to types

Reply:
Disregard me, this is actually fine here, types module contains shared types so this is a better home for it